src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -243,12 +288,9 @@ public static Vector4 Over(Vector4 destination, Vector4 source, Vector4 blend)
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     public static Vector256<float> Over(Vector256<float> destination, Vector256<float> source, Vector256<float> blend)
     {
-        const int blendAlphaControl = 0b_10_00_10_00;
-        const int shuffleAlphaControl = 0b_11_11_11_11;
-
         // calculate weights
-        Vector256<float> sW = Avx.Shuffle(source, source, shuffleAlphaControl);
-        Vector256<float> dW = Avx.Shuffle(destination, destination, shuffleAlphaControl);
+        Vector256<float> sW = Avx.Shuffle(source, source, ShuffleAlphaControl);
+        Vector256<float> dW = Avx.Shuffle(destination, destination, ShuffleAlphaControl);
         Vector256<float> blendW = Avx.Multiply(sW, dW);
 
         Vector256<float> dstW = Avx.Subtract(dW, blendW);
```

Comment:
You should be able to build a `MultiplyAddNegatedEstimate` helper using `Fma.MultiplyAddNegate` since that does `-(a * b) + c`

Reply:
I'd need [`Vector256.Negate<T>`](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.vector256.negate?view=net-7.0) for that wouldn't I?

Reply:
You'd use `Fma.MultiplyAddNegate(a, b, c)` and fallback to `Avx.Subtract(c, Avx.Multiply(a, b))` otherwise.

This is because `(x - y)` == `(-y + x)` and so you're just taking advantage of the specialized instruction `Fma` instruction that negates the multiplied result itself.

Reply:
Yep. I see that. In this instance I use `blendW` further down so it looks like I'd be better off keeping as is since I would be introducing duplicate multiplications.